src/hs_helper.c

```diff
@@ -254,12 +254,6 @@ static gboolean
 rspamd_rs_compile (struct hs_helper_ctx *ctx, struct rspamd_worker *worker,
 		gboolean forced)
 {
-	if (!(ctx->cfg->libs_ctx->crypto_ctx->cpu_config & CPUID_SSSE3)) {
-		msg_warn ("CPU doesn't have SSSE3 instructions set "
-				"required for hyperscan, disable hyperscan compilation");
-		return FALSE;
-	}
-
```

Comment:
We should leave this check for x86/x86_64 platform.

Reply:
Added back the removed sections with checks for __aarch64__

Reply:
Thanks!